restapi/src/main/java/io/stargate/web/restapi/dao/RestDBAccess.java

```diff
@@ -175,6 +173,50 @@ public Schema schema() {
     return dataStore.schema();
   }
 
+  public void authorizeDataRead(String keyspace, String table, SourceAPI sourceAPI)
```

Comment:
Added pass-through methods to encapsulate `authenticationSubject` (instead of giving accessor), to slightly simplify call patterns from resources.

Reply:
Again, seems we are complicating more than simplifying..